docs/deploying.md

```diff
@@ -101,7 +101,7 @@ Resources:
   FckNatAsgLaunchConfig:
     Type: AWS::AutoScaling::LaunchConfiguration
     Properties:
-      ImageId: ami-05b6d5a2e26f13c93
+      ImageId: ami-0591f971d15aec0ab
```

Comment:
This change might not be as relevant as AMI ID change from region to region. That said the original AMI is from (in `us-east-1`) is using version 1.2.0 (vs latest 1.2.1).

Perhaps adding a step in the deploy instructions above to change the AMI ID based on the region fck-nat is deployed in might be more adapted.